php/test/test_async.php

```diff
@@ -84,6 +84,44 @@ function dump($s) {
     }
 }
 
+
+// ### common language specific methods ###
```

Comment:
these added methods in py & php, are not yet used, because as said before, there is a PR of chain, and next PR's will be using these methods.